src/js/bootstrap-slider.js

```diff
@@ -472,12 +472,11 @@ const windowIsDefined = (typeof window === "object");
 
 				/* Create highlight range elements */
 				this.rangeHighlightElements = [];
-				if (Array.isArray(this.options.rangeHighlights) && this.options.rangeHighlights.length > 0) {
-					for (let j = 0; j < this.options.rangeHighlights.length; j++) {
-
+                var rangeHighlightsOpts = this.options.rangeHighlights;
```

Comment:
could you fix the spacing and declare this as a `const` instead of a `var` since the value is not being reassigned? thanks!